examples/26_2_spatial_mobilenet_mono.py

```diff
@@ -98,32 +98,47 @@
 
     rectifiedRight = None
     detections = []
-
     start_time = time.monotonic()
     counter = 0
-    fps = 0
     color = (255, 255, 255)
 
+    def frame_norm(frame, bbox):
```

Comment:
this is actually denormalization, and IMO this code is harder to understand than the previous one

Reply:
It follows the same convention as in the previous example. With this PR, I want to make examples use a common structure so that it's easier to later extend the examples or extract some common parts (this method can be one of them).